### PR TITLE
Canadian Brands

### PR DESCRIPTION
**Pull request by mparrault**
-Mucho Burrito, Jimmy the Greek, and Sunrise Records

### CODE REVIEW

File: brands/amenity/fast_food.json [hunk 1]
```diff
@@ -959,6 +959,18 @@
       "takeaway": "yes"
     }
   },
+  "amenity/fast_food|Jimmy the Greek": {
+    "countryCodes": ["ae", "ca"],
+    "tags": {
+      "amenity": "fast_food",
+      "brand": "Jimmy the Greek",
+      "brand:wikidata": "Q1689380",
```
Comment (bhousel) [inline brands/amenity/fast_food.json]: Should be Q17077817
(Q1689380 points to "Jimmy John's")

File: brands/amenity/fast_food.json [hunk 2]
```diff
@@ -1285,6 +1297,17 @@
       "takeaway": "yes"
     }
   },
+  "amenity/fast_food|Mucho Burrito": {
+    "countryCodes": ["ca", "gb", "us"],
+    "tags": {
+      "amenity": "fast_food",
+      "brand": "Mucho Burrito",
+      "brand:wikidata": "Q1689380",
```
Comment (bhousel) [inline brands/amenity/fast_food.json]: This one too .. Can you please fix the qids and let me know when this is ready for review?

File: brands/amenity/fast_food.json (hunk 1 above)
Comment (mparrault) [reply brands/amenity/fast_food.json]: fixed

File: brands/amenity/fast_food.json (hunk 2 above)
Comment (mparrault) [reply brands/amenity/fast_food.json]: fixed

**Comment by bhousel** [comment]
Thank you

**Pull request merged**